src/pcrlock/pcrlock.c

```diff
@@ -68,6 +68,7 @@ static bool arg_force = false;
 static BootEntryTokenType arg_entry_token_type = BOOT_ENTRY_TOKEN_AUTO;
 static char *arg_entry_token = NULL;
 static bool arg_varlink = false;
+static unsigned abbrev_hash = 7;
```

Comment:
we have this rule that all global variables that are directly parsed from our command line (or env block) are prefixed with `arg`. Please do so here too.

Please use `size_t`, since string lengths are usually handled in size_t. (see what strlen() returns). Not that this matters, but it's the choice of least surprise I would say.

Reply:
not a fan of unnecessary abbreviations, hence maybe call this `arg_abbreviate_hash`